Slim/Plugin/ExtendedBrowseModes/Settings.pm

```diff
@@ -32,7 +33,7 @@ use constant AUDIOBOOKS_MENUS => [{
 	weight  => 15,
 	enabled => 0,
 }];
-
+my $log   = logger('prefs');
```

Comment:
That's not really used here, is it? You could actually remove the import as well.

Reply:
A leftover from debugging.